raiden/api/python.py

```diff
@@ -213,7 +213,6 @@ def token_network_register(
 
         try:
             registry = self.raiden.chain.token_network_registry(registry_address)
-            # LEFTODO: Supply a proper block id
             return registry.add_token(
                 token_address=token_address,
                 given_block_identifier='latest',
```

Comment:
What about the block hash in this file? Will it be done in a follow up PR?

Reply:
Yep, as we discussed will do it in https://github.com/raiden-network/raiden/pull/3515